examples/rethinkdb/v1beta3/README.md

```diff
@@ -0,0 +1,106 @@
+RethinkDB Cluster on Kubernetes
+==============================
+
+Setting up a [rethinkdb](http://rethinkdb.com/) cluster on [kubernetes](http://kubernetes.io)
+
+**Features**
+
+ * Auto configuration cluster by querying info from k8s
+ * Simple
+
+Quick start
+-----------
+
+**Step 1**
+
+antmanler/rethinkdb will discover peer using endpoints provided by kubernetes_ro service,
+so first create a service so the following pod can query its endpoint
+
+```shell
+kubectl create -f driver-service.yaml
+```
+
+check out:
+
+```shell
+$kubectl get se
+NAME                LABELS              SELECTOR                  IP                  PORT
+rethinkdb-driver    db=influxdb         db=rethinkdb              10.241.105.47       28015
+```
+
+**Step 2**
+
+start fist server in cluster
+
+```shell
+kubectl create -f rc.yaml
+```
+
+Actually, you can start servers as many as you want at one time, just modify the `replicas` in `rc.ymal`
```

Comment:
Should be "Actually, you can start as many servers as you require, all at once.  Just modify the `replicas` field in `rc.ymal` appropriately."